Formula/rcon.rb

```diff
@@ -0,0 +1,21 @@
+class Rcon < Formula
+  desc "Source RCON client for shell"
+  homepage "https://github.com/n0la/rcon"
+  url "https://github.com/n0la/rcon/archive/0.5.tar.gz"
+  sha256 "28155fca4de607e7ea7101985178aa7f29ea9de624760e4cbcd2746f7f7f6e65"
+
+  depends_on "check" => :build
+  depends_on "cmake" => :build
+  depends_on "glib" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "cmake", ".", "-DINSTALL_BASH_COMPLETION=OFF", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "No host and/or port specified", pipe_output("#{bin}/rcon").chomp
```

Comment:
This takes a second argument as the input. Do you mean `shell_output`?

Reply:
yeah, `stderr` to be exact.

Reply:
`shell_output` with a ` 2>&1` is the best way

Reply:
thanks!